spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -431,6 +434,23 @@ class Graph[T: ClassTag](val inputs : Seq[ModuleNode[T]],
       }
     }
   }
+
+  /**
+   * Save current model graph to log folder, which can be display in tensorboard by running
+   *   tensorboard --logdir logPath
+   * @param logPath
+   * @return
+   */
+  def saveToLog(logPath: String): this.type = {
```

Comment:
What not just use `saveToTensorboard`? 
I don't know what `saveToLog` will do unless I see the comments.